doc/Enable-GEOS-and-Proj4-on-Heroku.md

```diff
@@ -79,6 +83,8 @@ heroku plugins:install heroku-repo
 heroku repo:purge_cache
 ```
 
+**Note** This option will NOT work with [rgeo-proj4](https://github.com/rgeo/rgeo-proj4). 
```

Comment:
The buildpack's readme explicitely says that it works with gdal and proj for rgeo. Hence I think this should be an issue in the buildpack's repo (or a PR!).

Also one tiny change: if you could use the same way of adding links that the doc already use:

```suggestion
**Note** This option will NOT work with [rgeo-proj4]. 
```

and at the end of the doc:

```md
[rgeo-proj4]: https://github.com/rgeo/rgeo-proj4
```

Reply:
I think the issue is that with the update to v3 of RGeo-Proj4, it depends on PROJ 6.2+, but the buildpack installs PROJ 5.2 by default. It seems like there's a way to install a different version though. I don't use Heroku so I'm not familiar with the process, but that should get it to work.

Reply:
@BuonOmo I update to using the style of links you suggested. It's neat! I didn't know markdown could do that.

Reply:
@BuonOmo & @keithdoggett - I'll admit, after some initial difficulties with the buildpack I gave up on it. However, I totally _could_ work. I've updated the comment to reflect that.